canvas_modules/common-canvas/__tests__/common-properties/components/table-toolbar-test.js

```diff
@@ -851,117 +901,127 @@ describe("TableToolbar multi select edit button works correctly", () => {
 
 		// select the first row in the table
-		let tableRows = tableUtils.getTableRows(summaryPanel);
+		let tableRows = tableUtilsRTL.getTableRows(summaryPanel);
 		expect(tableRows).to.have.length(4);
-		tableUtils.selectCheckboxes(summaryPanel, [0]);
-		summaryPanel = propertyUtils.openSummaryPanel(wrapper, "SLE_mse_panel");
+		tableUtilsRTL.selectCheckboxes(summaryPanel, [0]);
+		summaryPanel = propertyUtilsRTL.openSummaryPanel(wrapper, "SLE_mse_panel");
 
 		// verify that the "Edit" button is not present in table toolbar because you selected only 1 row
-		let tableToolbar = wrapper.find("div.properties-table-toolbar");
-		let multiSelectEditButton = tableToolbar.find("button.properties-action-multi-select-edit");
-		expect(multiSelectEditButton).to.have.length(0);
+		let tableToolbar = container.querySelector("div.properties-table-toolbar");
+		let multiSelectEditButton = tableToolbar.querySelector("button.properties-action-multi-select-edit");
+		expect(multiSelectEditButton).to.not.exist;
 
 		// select four rows in the table
-		tableUtils.selectCheckboxes(summaryPanel, [1, 2, 3]);
-		summaryPanel = propertyUtils.openSummaryPanel(wrapper, "SLE_mse_panel");
+		tableUtilsRTL.selectCheckboxes(summaryPanel, [1, 2, 3]);
+		summaryPanel = propertyUtilsRTL.openSummaryPanel(wrapper, "SLE_mse_panel");
 
 		// verify that the "Edit" button is present in table toolbar
-		tableToolbar = wrapper.find("div.properties-table-toolbar");
-		multiSelectEditButton = tableToolbar.find("button.properties-action-multi-select-edit");
-		expect(multiSelectEditButton).to.have.length(1);
+		tableToolbar = container.querySelector("div.properties-table-toolbar");
+		multiSelectEditButton = tableToolbar.querySelector("button.properties-action-multi-select-edit");
+		expect(multiSelectEditButton).to.exist;
 
-		// Click the multiSelectEditButton in table toolbar
-		multiSelectEditButton.simulate("click");
+		// Click the multiSelectEditButton in table toolbars
+		fireEvent.click(multiSelectEditButton);
 
 		// A new panel opens which shows editable columns
-		let wideFlyoutPanel = wrapper.find(".properties-wf-children");
-		let editableColumns = wideFlyoutPanel.find(".properties-editstyle-inline").find(".properties-ctrl-wrapper");
+		let wideFlyoutPanel = container.getElementsByClassName("properties-wf-children");
+		let editableColumns = [];
+		for (let i = 0; i < wideFlyoutPanel.length; i++) {
+			const col = wideFlyoutPanel[i].querySelector(".properties-ctrl-wrapper");
+			if (col.parentElement.className.includes("properties-control-panel")) {
+				editableColumns.push(col);
+			}
+		}
 		expect(editableColumns).to.have.length(2); // Animals column has edit_style: "subpanel". Can't edit from selectedEditCells.
 
 		// Set 44 for Integer field
-		let integerNumber = editableColumns.at(0).find("input");
-		integerNumber.simulate("change", { target: { value: "44" } });
-
+		let integerNumber = within(editableColumns[1]).getAllByRole("spinbutton")[0];
```

Comment:
In Enzyme it is testing first item `editableColumns.at(0)` but here I can see `editableColumns[1]` are these two same ?

Reply:
yes these are the same, since I'm getting `editableColumns[1].getAllByRole("spinbutton")[0]`